projects/novo-elements/src/elements/form/ControlGroup.ts

```diff
@@ -170,6 +157,10 @@ export class NovoControlGroup implements AfterContentInit, OnChanges {
     }
   }
 
+  onChange(change: any): void {
+    this.change.emit(this);
+  }
```

Comment:
do we need to emit `this`? why not just emit the change?

Reply:
the change is the element itself (the picker, dropdown, date...). It doesn't have the index